### PR TITLE
serializers: rename id_ to uuid and add _collections to publich schemas

### PR DESCRIPTION
**Pull request by monaawi**
* Renamed id_ to uuid
* INSPIR-2572
* Added _collections to public schemas. This is needed for the record importer by url
* INSPIR-2575

### CODE REVIEW

**Comment by michamos** [comment]
not making `_collections` visible is OK, as a record might also be in a hidden collection the user has no access to, so he shouldn't see those hidden collections mentioned. Why is this needed?

**Comment by drjova** [comment]
```s/publich/publish```

File: tests/integration/records/serializers/test_json.py [hunk 1]
```diff
@@ -287,9 +291,11 @@ def test_literature_json_ui_v1_response(api_client, db, create_record):
     response_status_code = response.status_code
     response_data = json.loads(response.data)
     response_data_metadata = response_data["metadata"]
+    response_data_uuid = response_data["uuid"]
 
     assert expected_status_code == response_status_code
     assert expected_result_metadata == response_data_metadata
+    assert expected_uuid == response_data_uuid
```
Comment (drjova) [inline tests/integration/records/serializers/test_json.py]: could we also add the ``id``, ``created``, ``updated`` as well?  I think they are missing from the assertions.

**Comment by monaawi** [comment]
> not sure making `_collections` visible is OK, as a record might also be in a hidden collection the user has no access to, so he shouldn't see those hidden collections mentioned. Why is this needed?

As discussed IRL, I removed this part for now and will do it separately without making the _collections public anymore

**Pull request merged**